exercises/reverse-string/example.rkt

```diff
@@ -0,0 +1,11 @@
+#lang racket
+
+(provide (contract-out
+          [my-reverse (string?  . -> . string?)]))
+
+(define (my-reverse s)
+  (let loop ([v (string->list s)]
+             [answer '()])
+    (if (empty? v)
+      (list->string answer)
+      (loop (cdr v) (cons (car v) answer)))))
```

Comment:
It might be none of my business, but `foldl` gives us a quick way to do this without conditionals:
``` racket
(define (my-reverse str)
  (list->string (foldl cons '() (string->list str))))
```

Reply:
That is *much* better than my original implementation! Please consider it to be your business :)

I added your suggestion as https://github.com/exercism/racket/pull/129, I hope that you don't mind.